README.md

```diff
@@ -1,2 +1,83 @@
 # Reps-Mobile
-reps.mozilla.org data on mobile
+
+<p align="center">
+  <a href="https://flutter.dev/">
+    <img alt="Flutter" src="https://flutter.dev/assets/flutter-lockup-4cb0ee072ab312e59784d9fbf4fb7ad42688a7fdaea1270ccf6bbf4f34b7e03f.svg" width="170" />
+  </a>
+</p>
+<h1 align="center">
+ Flutter Starter 
+</h1>
+
+Kick off your project with this boilerplate. This starter will help to install  the main flutter configuration files along with the brief introduction to Mozilla Reps mobile application.
```

Comment:
This is not a boilerplate for Flutter, since this is an app. Please remove this line.

Reply:
done!

Reply:
@arps18 please, this is not a boilerplate and we are not suggesting new contributors to kickstart their *own* project. This is an app for contributors to test the app themselves wether or not they have any knowledge of flutter at all. Here the primary focus is not the technology we are using but the app. I suggest you remove the line.

Reply:
Bro, I am extremely sorry. Initially, I removed it but due to some issue maybe it was undone and was overlooked by me. I assure you that it will be removed and extremely sorry for the trouble.

Reply:
Done!